src/aws_utils/cloudwatch_utils.rs

```diff
@@ -77,12 +81,20 @@ pub struct MetricBuilder {
     value: f64,
 }
 
+// TODO: TxStatus type metrics => TxStatus(u32)
```

Comment:
can we move TODOs into linear tix?

Reply:
can create linear tix for sure, but I still prefer to have TODOs in code as comments to remind myself from time to time